tests/Header.test.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import Header from '../src/ui/Header';
+import { Navigator, ReaderState } from '../src/types';
+
+test('render header bar', () => {
+  const readerState = {} as ReaderState;
+  const navigator = {} as Navigator;
+
+  render(<Header readerState={readerState} navigator={navigator} />);
+
+  // Should also test the href once everthings been implemented in the furture
```

Comment:
Additional tests:  
1) Mount and pass a custom thing headertext and icon and make sure it shows up.

Reply:
I added a test using `data-testid` for the icon b/c I can't target the Icon. Would love to know if there are other ways to go for this.